src/cpprealm/internal/network/network_transport.cpp

```diff
@@ -106,13 +106,16 @@ namespace realm::internal {
         DefaultSocket socket{service};
         socket.connect(ep);
 
+#if REALM_INCLUDE_CERTS
+        m_ssl_context.use_included_certificate_roots();
+#endif
```

Comment:
```suggestion
        m_ssl_context.use_default_verify();
#if REALM_INCLUDE_CERTS
        m_ssl_context.use_included_certificate_roots();
#endif
```
The built-in websocket client calls both.